doc/man7/OSSL_PROVIDER-default.pod

```diff
@@ -191,6 +191,8 @@ The OpenSSL default provider supports these operations and algorithms:
 
 =item SM2
 
+=item ML-DSA, see L<EVP_SIGNATURE-ML-DSA(7)>
```

Comment:
Hmm. Here (and in key management) we refer to it as ML-DSA. But in the encoders/decoders section they are ML-DSA-44/65/87. Since they are actually separate key types shouldn't we be listing all 3 names here?